DSharpPlus/Entities/Channel/Message/DiscordMessageBuilder.cs

```diff
@@ -198,13 +207,14 @@ public DiscordMessageBuilder WithFiles(Dictionary<string, Stream> files, bool re
         /// <param name="messageId">The ID of the message to reply to.</param>
         /// <param name="mention">If we should mention the user in the reply.</param>
         /// <returns></returns>
-        public DiscordMessageBuilder WithReply(ulong messageId, bool mention = false)
+        public DiscordMessageBuilder WithReply(ulong messageId, bool mention = false, bool failOnInvalidReply = false)
```

Comment:
There should also be a description for the parameter here

Reply:
Oops